yus

Dotlezz opened the pull request.

Commit:
Delete Modules directory

Commit:
Delete Main.cs

Commit:
Delete Properties directory

Commit:
Delete CustomLoadAudio.sln

Commit:
Delete CustomLoadAudio.csproj

Commit:
Add Source Code

Commit:
Add files via upload

Commit:
let rider do funny format

Commit:
Add files via upload

Commit:
Update README.md

Commit:
Awesome audio that I use

Mezque reviewed (changes requested):
- keep Properties/AssemblyInfo.cs 
- add back trolly mod author colour
- the choice to load audio from User data or a specific file path provided by the user

Mezque commented:
Fyi the download link part of MelonInfo assembly attributes does not really do anything

Mezque committed:
Pr changes

1. My original idea to make the toggle to use a custom path or user data was kind of stupid, so I didn't make that.

2. Readded back Assembly info

3. Made mod author purple again lol

4. Set the default location up to be userdata\CustomLoadAudio

5. Created the folder Userdata\CustomLoadAudio if its none existent.

Co-Authored-By: Dotlezz <103538714+dotlezz@users.noreply.github.com>

Commit:
Update Source Code

Commit:
Update Source Code - Let Rider monky it

Commit:
dont think so

Commit:
nah

Commit:
yeah no

Commit:
Update

Commit:
Delete CustomLoadAudio.csproj

Mezque commented:
there

Mezque commented:
@Dotlezz its closed and merged now

Mezque commented:
<img width="1235" alt="image" src="https://user-images.githubusercontent.com/31026406/167210846-9848aa41-d7ef-4128-9892-d24ed313c8bd.png">
github should rope its neck lol
